examples/platformer/platformer.json

```diff
@@ -14,9 +14,9 @@
     "pixelsRounding": false,
     "projectUuid": "17fcdaeb-d145-42e7-8540-aab134538ef2",
     "scaleMode": "linear",
-    "sizeOnStartupMode": "",
+    "sizeOnStartupMode": "adaptWidth",
     "useExternalSourceFiles": false,
-    "version": "1.1.9",
+    "version": "1.1.10",
```

Comment:
From 1.1.9 we must move to 1.2.0

Reply:
I don't mind changing to a higher minor number, but semantic versioning doesn't require it.

Reply:
Yep 1.1.10 is the logical next number @Bouh